curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-a-gradual-css-linear-gradient.md

```diff
@@ -10,7 +10,7 @@ forumTopicId: 301047
 <section id='description'>
 Applying a color on HTML elements is not limited to one flat hue. CSS provides the ability to use color transitions, otherwise known as gradients, on elements. This is accessed through the <code>background</code> property's <code>linear-gradient()</code> function. Here is the general syntax:
 <code>background: linear-gradient(gradient_direction, color 1, color 2, color 3, ...);</code>
-The first argument specifies the direction from which color transition starts - it can be stated as a degree, where 90deg makes a vertical gradient and 45deg is angled like a backslash. The following arguments specify the order of colors used in the gradient.
+The first argument specifies the direction from which color transition starts - it can be stated as a degree, where 90deg makes a gradient from left to right and 45deg is angled like a backslash. The following arguments specify the order of colors used in the gradient.
```

Comment:
@steviehailey You have made a great catch here!  I would make one small adjustment to clarify that `90deg` creates a `horizontal` gradient.  For example:
```suggestion
The first argument specifies the direction from which color transition starts - it can be stated as a degree, where `90deg` makes a horizontal gradient (from left to right) and `45deg` is angled like a backslash. The following arguments specify the order of colors used in the gradient.
```
If you agree with this suggestion, you can simply click the `Commit suggestion` button and add an applicable commit message for a second commit.

Reply:
@RandellDawson should we be using backticks for code, or wrapping them around `<code>` tags? I was going to suggest `<code>` tags, but maybe something has changed and we need to [update the contributor docs](https://contribute.freecodecamp.org/#/how-to-work-on-coding-challenges?id=formatting-challenge-text).

Reply:
Hey @erictleung I'd agree those docs do suggest I should swap the back tick for code tags
> References to code parts (i.e. function, method or variable names) should be wrapped in <code> tags
Will push another commit...

Reply:
@erictleung Just to mention it: The docs do say
> Note: The equivalent Markdown should be used, where applicable, in place of HTML tags.

Reply:
@Sky020 ah, thanks! I didn't read down the page far enough 😅 Maybe we can reword the docs to make it less confusing then, but that's another issue beyond the scope of this PR